mods/ra/rules/structures.yaml

```diff
@@ -1274,12 +1274,14 @@ AFLD:
 	Buildable:
 		Queue: Building
 		BuildPaletteOrder: 130
-		Prerequisites: dome, ~structures.soviet, ~techlevel.medium
+		Prerequisites: dome, ~structures.soviet, ~techlevel.medium, ~!structures.ukraine
```

Comment:
The parabombs are tied to `aircraft.ukraine`, so I think it would be better to use that here than `structures.ukraine`.